test/standalone/test-config-json.js

```diff
@@ -23,7 +23,7 @@ var assert = require('assert');
 
 // Fixtures configuration:
 // { logLevel: 4, stackTraceLimit: 1 };
-process.env.GCLOUD_TRACE_CONFIG = path.join('test', 'fixtures', 'test-config.json');
+process.env.GCLOUD_CONFIG = path.join('test', 'fixtures', 'test-config.json');
```

Comment:
`GCLOUD_CONFIG` is too generic a name. Could be confused with the gcloud cli, or with the `gcloud` npm module.